digdag-standards/src/main/java/io/digdag/standards/operator/param/RedisParamServerClient.java

```diff
@@ -0,0 +1,132 @@
+package io.digdag.standards.operator.param;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Optional;
+import com.google.common.base.Throwables;
+import io.digdag.spi.Record;
+import io.digdag.spi.ValueType;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.Transaction;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+/*
+    This class is not Threadsafe, but this class is
+    expected to be bound as NOT singleton in guice container.
+    see ParamServerModule
+ */
+public class RedisParamServerClient
+        implements ParamServerClient
+{
+    private final ObjectMapper objectMapper;
+    private Jedis connection;
+    private Map<String, String> msetTarget = new HashMap<>();
+
+    public RedisParamServerClient(ParamServerClientConnection connection, ObjectMapper objectMapper)
+    {
+        this.connection = (Jedis) connection.get();
+        this.objectMapper = objectMapper;
+    }
+
+    @Override
+    public Optional<Record> get(String key, int siteId)
+    {
+        if (connection == null) {
+            throw new IllegalStateException("Connection has already closed");
+        }
+
+        // "{value: {value: 1}, value_type: 0}"
+        String rawResult = connection.get(formattedKey(key, siteId));
+        if (rawResult == null) { return Optional.absent(); }
+
+        JsonNode json;
+        try {
+            json = objectMapper.readTree(rawResult);
+        }
+        catch (IOException e) {
+            throw Throwables.propagate(e);
+        }
+        JsonNode value = json.get("value");
+        int valueType = json.get("value_type").asInt();
```

Comment:
I think its' ok because this is temoprary variable assign, and `valueType` is converted to enum object soon https://github.com/treasure-data/digdag/blob/10c6fcbefc4fa4bababece9108ca6aac39017494/digdag-standards/src/main/java/io/digdag/standards/operator/param/RedisParamServerClient.java#L60